magda-web-client/src/Components/Dataset/View/DatasetPageSuggestForm.js

```diff
@@ -85,27 +85,36 @@ export default class DatasetPageSuggestForm extends React.Component {
         };
         const formProps = {
             title: false,
+            description:
+                "Your question will be sent to the publishers of the data: " +
+                this.props.contactPoint,
             namePlaceHolder: "Dorothy Hill",
             emailPlaceHolder: "dorothyhill@example.com",
             textAreaPlaceHolder:
                 "Ask a question or report a problem about this dataset.",
             textAreaLabel: "What would you like to ask about this dataset?",
-            successHeader: "Your request has been submitted!"
+            successHeader:
+                "Your request has been submitted to: " +
+                this.props.contactPoint +
```

Comment:
Currently, this request will be emailed to one single recipient (email address), right?
It seems the text here suggests different behaviour?
Or we have plan to adjust the logic of sending emails?

Reply:
I've rephrased this, hopefully it will be clearer that there is only one publisher.
Requests will only be sent to the publisher of the dataset, so only one email address.

Reply:
Hey Jacky,
The desired functionality is to not send to the default recipient (data@digital.gov.au) when there is no contact email provided.

I believe if we [update lines 187-194](https://github.com/magda-io/magda/blob/373c89f074d8aa99a4f6b1dc385722198b3531ec/magda-correspondence-api/src/createApiRouter.ts#L187-L194) in the `/public/send/dataset/:datasetId/question` endpoint, it would achieve this functionality.

We can use this

```
                const emails = [contactPointEmail, datasetPublisherEmail].filter(...)

                if (!validEmail) {
                    return res.status(404).json({
                        status: "Failed to send",
                        error:
                            "Could not find a contact email for the publisher"
                    });
                }

                const recipient = emails[0];
```
I believe the contactPointEmail corresponds to this.props.contactPoint?